test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -311,72 +306,138 @@ get_stat(WT_CURSOR *cursor, int stat_field, int64_t *valuep)
     error_check(cursor->get_value(cursor, &desc, &pvalue, valuep));
 }
 
-int
-main(int argc, char *argv[])
+static void
+run_restore(const std::string &home, const std::string &source, const int64_t thread_count,
+  const int64_t collection_count, const int64_t op_count, const bool background_thread_mode,
+  const int64_t verbose_level, const bool first)
 {
-    /* Set the program name for error messages. */
-    const std::string progname = testutil_set_progname(argv);
-    /* Set the tracing level for the logger component. */
-    logger::trace_level = LOG_TRACE;
-
     /* Create a connection, set the cache size and specify the home directory. */
-    /* FIXME-WT-13825: Set max_threads to non zero once extent list concurrency is implemented. */
-    // TODO: Make verbosity level configurable at runtime.
+    const std::string verbose_string =
+      verbose_level == 0 ? "" : "verbose=[fileops:" + std::to_string(verbose_level) + "]";
     const std::string conn_config = CONNECTION_CREATE +
-      ",live_restore=(enabled=true,threads_max=0,path=\"" + SOURCE_DIR +
-      "\",debug=(fill_holes_on_close=true)),cache_size=1GB,verbose=[fileops:2],statistics=(all),"
-      "statistics_log=(json,on_close,wait="
+      ",live_restore=(enabled=true,threads_max=" + std::to_string(thread_count) + ",path=\"" +
+      source + "\"),cache_size=1GB," + verbose_string +
+      ",statistics=(all),statistics_log=(json,on_close,wait="
       "1)";
 
-    logger::log_msg(LOG_TRACE, "arg count: " + std::to_string(argc));
-    bool fresh_start = false;
-    bool background_debug = false;
-    if (argc > 1 && argv[1][1] == 'f') {
-        fresh_start = true;
-        logger::log_msg(LOG_WARN, "Started in -f mode will clean up existing directories");
-        // Live restore expects the source directory to exist.
-        testutil_recreate_dir(SOURCE_DIR);
-        testutil_remove("WT_TEST");
-    } else if (argc > 1 && argv[1][1] == 'b') {
-        /*
-         * Operate in background thread debug mode, in this case we turn off CRUD operations and
-         * just wait until the relevant statistic is set then exit.
-         *
-         * Right now background_debug and fresh_start are mutually exclusive.
-         */
-        background_debug = true;
-    }
-    // We will recreate this directory every time, on exit the contents in it will be moved to
-    // WT_LIVE_RESTORE_SOURCE/.
-    testutil_recreate_dir("WT_TEST");
+    //,debug=(fill_holes_on_close=true)
```

Comment:
remove?

Reply:
cc: @ajmorton do you think we should turn this into another command line option? So it can be turned on and off at will?

Reply:
I wouldn't bother. We really only need it until the background thread concurrency control is merged. At that point we can replace it entirely with 
```
while migration.status != DONE:
    sleep(1)
```

Reply:
34c3f85cd